### ISSUE TITLE
React Grid Selection Example

### ISSUE THREAD

**Issue by radomirchev** [IgniteUI/igniteui-xplat-examples#397]
https://staging.infragistics.com/products/ignite-ui-react/react/components/grids/grid/selection

MultipleCascade is not doing anything

![image](https://github.com/IgniteUI/igniteui-xplat-examples/assets/52001020/dfa6da75-95bb-4999-80a8-bcaac260d1f2)

**Comment by MayaKirova** [comment]
@radomirchev @dkamburov This is a generated sample, so it shows all possible options for the property:
https://www.infragistics.com/products/ignite-ui-angular/docs/typescript/latest/variables/GridSelectionMode-1.html

`multipleCascade` is a valid option for all grids, however it has no effect if the grid has no hierarchy (is not a tree grid or a hierrachical grid). That's why it does nothing.

There isn't an option to exclude some of enum values for a generated sample, so if you want this option to not show for the base grid we would need to make this a custom sample (not xplat generated, one for react/wc/blazor).

**Comment by dkamburov** [comment]
@ChronosSF FYI

**Comment by tishko0** [comment]
@ChronosSF Can you please confirm we will be making these samples?

**Comment by ChronosSF** [comment]
No, we shouldn't. What we should do is add functionality to the editor to have overridable options for each grid type and export samples according to those.

**Comment by MayaKirova** [comment]
@ChronosSF Another option would be to remove this as an available option for the igxGrid (in other words - refactor the enum types) as it does nothing in it.

**Comment by dkamburov** [comment]
@radomirchev @ChronosSF should we simply close that one with no action?
